packages/js/components/src/experimental-select-control/select-control.tsx

```diff
@@ -123,7 +123,7 @@ function SelectControl< ItemType = DefaultItemType >( {
 	className,
 	disabled,
 	inputProps = {},
-	suffix = <SuffixIcon icon={ search } />,
+	suffix = <SuffixIcon icon={ chevronDown } />,
```

Comment:
Now that it is a chevron it makes me want to click on the `suffix` in order to open the menu, but at the moment it seems to do nothing when I click on the `suffix`.

Reply:
Seems to be a regression introduced in https://github.com/woocommerce/woocommerce/pull/37319.

Updated in https://github.com/woocommerce/woocommerce/pull/38052/commits/b9ea93ecfe26ff10e5331d5c620b3494a4ce0548